<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardSideWidget</class>
 <widget class="QWidget" name="WizardSideWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelLiveOffline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Live-/OfflineGame</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOfflineGeneral">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>General options</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOfflineGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game options</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOfflineResult">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoose">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOffline">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OfflineGame</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLive">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LiveGame</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLiveGeneral">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>General options</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLiveGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game options</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLiveSummary">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
